silhouette/app/com/mohiva/play/silhouette/impl/providers/SocialProvider.scala

```diff
@@ -165,6 +165,14 @@ trait SocialProfile {
   def loginInfo: LoginInfo
 }
 
+trait IdentifiedSocialProfile extends SocialProfile {
```

Comment:
@akkie I needed to subclass CommonSocialProfile ... but I couldn't as it is a case class and not a trait :-( The only choice was then introduce a trait that CommonSocialProfiles implements (and thus the IdentifiedSocialProfile) and have the new Auth0SocialProfile implementing it. Do you see a better way to get the same result?